src/style.css

```diff
@@ -1 +1,107 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  min-width: 1024px;
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  --height: 60px;
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.blockHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.logo:hover {
+  outline: 2px solid #00acdc;
```

Comment:
You could have moved this color to a variable (as I described above), and then used it everywhere you need it

Reply:
So it will look like this
```
outline: 2px solid var(--color);
```

Reply:
So it will look like this
```
outline: 2px solid var(--your-color);
```